search/search_quality/assessment_tool/main_model.hpp

```diff
@@ -25,9 +25,13 @@ class MainModel : public Model
   void OnSampleSelected(int index) override;
```

Comment:
Why? Open() just overrides a pure abstract method from model.hpp, and model.hpp includes string.